webapp/locales/de.json

```diff
@@ -484,8 +485,9 @@
         },
         "teaserImage": {
             "cropperConfirm": "Bestätigen"
-        },
-        "languageSelectText": "Sprache wählen"
+        },        
+        "inappropriatePicture" : "Dieses Bild kann für einige Menschen unangemessen sein.",
+        "inappropriatePictureText" : "Wann soll ein Foto versteckt werden"
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-12-18T10:02:24Z" title="Wednesday, December 18th 2019, 11:02:24 am +01:00">Dec 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -484,8 +485,9 @@
         },
         "teaserImage": {
             "cropperConfirm": "Bestätigen"
-        },
-        "languageSelectText": "Sprache wählen"
+        },        
+        "inappropriatePicture" : "Dieses Bild kann für einige Menschen unangemessen sein.",
+        "inappropriatePictureText" : "Beitragsbild wird Verschwommen dargestellt"
```


one last, tiny change – since this is the link text that leads to the FAQ, I would suggest something like:
```suggestion
        "inappropriatePictureText" : "Wann soll ein Foto versteckt werden"
```

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-12-18T11:20:52Z" title="Wednesday, December 18th 2019, 12:20:52 pm +01:00">Dec 18, 2019</time>_

---


ok